FaliJedan/FaliJedan.Domain/Repositories/Implementations/EventRepository.cs

```diff
@@ -22,13 +22,14 @@ public EventRepository(FaliJedanContext context)
         public bool AddEvent(Event eventToAdd)
         {
             if (eventToAdd.CurrentNumberOfPlayers < 1 || eventToAdd.TargetNumberOfPlayers < 2 ||
-                eventToAdd.DateOfEvent == null || eventToAdd.StartTime == null || eventToAdd.GeoLocation == null)
+                eventToAdd.DateOfEvent == null || eventToAdd.StartTime == null || eventToAdd.LocationLongitude == null || eventToAdd.LocationLatitude == null)
                 return false;
```

Comment:
This If statement is a bit clunky to read, Imo it is better for readability to make it one line per expression

i.e.

```suggestion
                if 
                (
                    eventToAdd.CurrentNumberOfPlayers < 1 ||
                    eventToAdd.TargetNumberOfPlayers < 2 ||
                    eventToAdd.DateOfEvent == null ||
                    eventToAdd.StartTime == null ||
                    eventToAdd.LocationLongitude == null ||
                    eventToAdd.LocationLatitude == null
                )
                return false;
```

Maybe even move all validations to a specific class designed to validate Entities for inputs
Also, just for info, validation for CurrentNumberOfPlayers >= TargetNumberOfPlayers